### PR TITLE
fix: remove _meta on all nodes when debug option is falsy

### LINKED ISSUE #70: Remove _meta on all nodes when debug option is falsy

### PR DESCRIPTION
**Pull request by ccalbaugh**
Fixes #70

### CODE REVIEW

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>The README has not been updated. Please update the README.</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  Branch being merged does not follow [Git Flow](http://nvie.com/posts/a-successful-git-branching-model/)
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Could not find any browser results.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 2e665be20143979569e5ef2a8614ae495ae5b600
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+### 0.12.4
+* Ensure _meta is stripped from filterOnly nodes / nodes without valid context if debug option is falsy
```
Comment (rudolph9) [inline CHANGELOG.md:2]: nit: missing a new line after this section in the changelog.

**Review by rudolph9 — APPROVED** [review]
I don't totally understand why we're making this change so it would probably be good to get someone else's approval who has more context but the implementation looks good.

File: CHANGELOG.md (hunk 1 above)
Comment (ccalbaugh) [reply CHANGELOG.md:2]: Thanks I'll add some space

**Pull request merged**